ui/main/src/app/services/card.service.ts

```diff
@@ -34,6 +34,7 @@ import {
 } from '@ofActions/light-card.actions';
```

Comment:
Sonar is complaining that `import * as moment from 'moment';` is unused (even though I don't know why it considers it as a new code smell when you didn't touch it).

Reply:
This warning appear when sonar make the analyse on the merge in develop so it was not due to this PR but to another PR that was merged before.